ev3dev2/motor.py

```diff
@@ -1765,12 +1839,20 @@ def on_for_degrees(self, left_speed, right_speed, degrees, brake=True, block=Tru
         # therefore, t = d_l/v_l = d_r/v_r
         
         if degrees == 0 or (left_speed_native_units == 0 and right_speed_native_units == 0):
+            log.warning("{}: degrees {}, left_speed_native_units {}, right_speed_native_units {}, robot will not move"
+                .format(self, degrees, left_speed_native_units, right_speed_native_units))
+            left_degrees = 0
+            right_degrees = 0
+
+        elif left_speed_native_units == right_speed_native_units:
```

Comment:
What's happening here? It looks like the logic is different now.

Reply:
I was just trying to simplify the code for the scenario where the left speed and right speed are equal. The end behavior should be the same as before

Reply:
OK, if the behavior should be the same I'll take your word for it